README.md

```diff
@@ -13,14 +13,18 @@ Once installed you can then use the `deepspeech` binary to do speech-to-text on
 
 ```bash
 pip install deepspeech
-deepspeech output_model.pb my_audio_file.wav alphabet.txt
+wget https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/deepspeech-0.1.0-models.tar.gz
```

Comment:
You could make that `wget -O https://github.com/mozilla/DeepSpeech/releases/download/v0.1.0/deepspeech-0.1.0-models.tar.gz | tar xvfz - ` to simplify even more

Reply:
And you should not repeat the same command line over and over, it's going to get out of sync soon. Rather it would be better to add a section , say "Downloading inference material" and refer to that section of the README everywhere it might be needed :)